packages/builder/src/components/design/settings/controls/ButtonActionEditor/actions/TriggerAutomation.svelte

```diff
@@ -24,7 +24,11 @@
     parameters
   }
   $: automations = $automationStore.automations
-    .filter(a => a.definition.trigger?.stepId === TriggerStepID.APP)
+    .filter(
+      a =>
+        a.definition.trigger?.stepId === TriggerStepID.APP &&
+        a.disabled !== true
```

Comment:
`!disabled` is usually preferred for flags, incase it were to be set to `1` for example